src/app/netsuite/sync-export/sync/sync.component.ts

```diff
@@ -50,9 +51,18 @@ export class SyncComponent implements OnInit {
       takeWhile((response) => response.results.filter(task => task.status === 'IN_PROGRESS'  && task.type === 'FETCHING_EXPENSES').length > 0, true)
     ).subscribe((res) => {
       if (!res.results.length) {
-        that.updateLastSyncStatus();
+        that.updateLastSyncStatus().subscribe((response) => {
+          that.isExpensesSyncing = false;
+          if (response[0].last_synced_at !== lastSyncedAt) {
+            that.snackBar.open('Import Complete');
+          } else {
+            const expenseState = that.expenseGroupSettings.expense_state.toLowerCase().replace('_', ' ');
+            that.snackBar.open(`No new expense group imported kindly check your fyle account to see if there are any expenses in the ${expenseState} state`, null, {
```

Comment:
No new expense groups were imported. Kindly check your Fyle account to see if there are any expenses in the paid state.
```suggestion
            that.snackBar.open(`No new expense groups were imported. Kindly check your Fyle account to see if there are any expenses in the ${expenseState} state`, null, {
```